faqs/galaxy/datatypes_fastq_format.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to format fastq data for tools that require .fastqsanger format
```

Comment:
title: How to format fastq data for tools that require .fastqsanger format?

question mark is missing.